src/vulkan-renderer/renderer.cpp

```diff
@@ -616,7 +598,7 @@ VkResult VulkanRenderer::create_frame_buffers() {
         VK_IMAGE_ASPECT_DEPTH_BIT | VK_IMAGE_ASPECT_STENCIL_BIT, VK_SAMPLE_COUNT_1_BIT, "Depth stencil image",
         swapchain->get_extent());
 
-    VkImageView attachments[4];
+    std::vector<VkImageView> attachments(4);
 
     if (multisampling_enabled) {
```

Comment:
Move the `depth_stencil` creation over the first `if (multisampling_enabled) {`  and merge both if cases.

Reply:
Done.